docker/dev/docker-compose.yml

```diff
@@ -10,24 +10,30 @@ services:
     ports:
       - '5433:5432'
     restart: unless-stopped
+    networks:
+      - testnet
     volumes:
       - pg-data:/var/lib/postgresql/data
+      - ./dbinit.sql:/docker-entrypoint-initdb.d/init.sql
```

Comment:
Should we have two `postgres` images (one for the wallet and one for Rafiki)?

Reply:
for dev I don't think we need to separate them